docs/rst/02-knora-ontologies/knora-base.rst

```diff
@@ -970,7 +863,7 @@ Knora's built-in support for “standoff” markup, which is stored
 separately from the text. This has some advantages over embedded markup
 such as XML. [#]_ While XML requires markup to have a hierarchical
 structure, and does not allow overlapping tags, standoff nodes do not
-have these limitations (Schmidt2016_). A standoff tag can be attached to
+have these limitations. [Schmidt2016]_ A standoff tag can be attached to
```

Comment:
Shouldn't the link be before the period?

Reply:
It's a footnote, so I just tried to format it like the other footnotes in that document. Apparently it was syntactically invalid before, but older versions of Sphinx never complained. When I upgraded Sphinx, it told me there was an error on that line. That's the only reason I changed it.